fast64_internal/sm64/sm64_objects.py

```diff
@@ -1601,6 +1607,19 @@ def sm64_obj_register():
 	bpy.types.Object.useBackgroundColor = bpy.props.BoolProperty(
 		name = 'Use Solid Color For Background', default = False)
 
+	bpy.types.Object.backgroundID = bpy.props.StringProperty(
+		name = 'Background Define', default = 'BACKGROUND_CUSTOM', 
+		description = 
+		'The background define that is passed into GEO_BACKGROUND\n'
+		'(ex. BACKGROUND_OCEAN_SKY, BACKGROUND_GREEN_SKY)')
+
+	bpy.types.Object.backgroundSegment = bpy.props.StringProperty(
```

Comment:
so as mentioned on Discord, to be clear, new properties are better added to [`SM64_ObjectProperties`](https://github.com/Fast-64/fast64/blob/d68b004da302f6d3080b0136f1dcdeceb7d01848/fast64_internal/sm64/sm64_objects.py#L1480)

then instead of `object.backgroundID` it would be `object.fast64.sm64.backgroundID` (as is defined in [`__init__.py`](https://github.com/Fast-64/fast64/blob/d68b004da302f6d3080b0136f1dcdeceb7d01848/__init__.py#L355) and in [`Fast64_ObjectProperties`](https://github.com/Fast-64/fast64/blob/d68b004da302f6d3080b0136f1dcdeceb7d01848/__init__.py#L268))